python/Ganga/GPIDev/Lib/Registry/PrepRegistry.py

```diff
@@ -14,14 +14,9 @@
 
 class PrepRegistry(Registry):
 
-    def __init__(self, name, doc, dirty_flush_counter=10, update_index_time=30, dirty_max_timeout=60, dirty_min_timeout=30):
-
-        super(PrepRegistry, self).__init__(name, doc, dirty_flush_counter, update_index_time, dirty_max_timeout, dirty_min_timeout)
-
-        self.releaseThread = threading.Thread(target=self.trackandRelease, args=())
-        self.releaseThread.daemon = True
-        self.releaseThread.start()
```

Comment:
OK, if people have stopped complaining about this then I'm happy to drop the code, We still don't release the prep metadata lock during running which I regard as a bug since this stopped working but if users aren't needing to launch multiple sessions it probably says something for how things have improved.